### DISCUSSION TITLE
Accound suspended with no notice

### DISCUSSION THREAD

**Discussion by Shutup-Bro** [community/community#132336] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My friend's account was suspended. He has no idea when it happened as NO EMAIL was sent to him, and he could not sign in so I have to ask this question for him. It is common for github to suspend accounts with no emails sent out? What could be some reasons for this suspension? He says he checked the TOS and did not find anything that he would possibly have violated.

**Comment by RuoyuLiu-f** [comment]
GitHub 暂停账户的情况并不常见，但确实会发生。账户被暂停的原因可能包括但不限于以下几点：

1.  **违反服务条款（TOS）**：即使用户认为自己没有违反，GitHub 的系统或人工审核可能会发现某些行为违反了 TOS。例如，发布恶意软件、侵犯版权、进行垃圾邮件活动等。
    
2.  **安全问题**：如果 GitHub 检测到账户存在安全风险，如账户被黑客攻击或存在异常登录行为，可能会暂时冻结账户以保护用户和平台的安全。
    
3.  **滥用行为**：包括但不限于滥用 GitHub 的功能（如过度使用 API）、滥用协作功能（如发送大量无用邀请）等。
    
4.  **法律要求**：GitHub 可能因为法律要求或政府命令而暂停账户。
    
5.  **误报**：有时，账户暂停可能是由于误报或系统错误导致的。
    

如果用户认为自己没有违反 TOS，可以尝试联系 GitHub 支持团队，提供相关信息和证据，请求复查账户状态。GitHub 通常会提供一个申诉流程，用户可以通过这个流程来解决账户被暂停的问题。

为了避免账户被暂停，用户应该仔细阅读并遵守 GitHub 的服务条款和社区指南，确保自己的行为符合平台规定。

**Comment by DoKM-Dev** [comment]
My account, the account of a fellow contributor of one of the projects i worked on and the account of a contributor of a similair project have all gotten suspended without notice half a day ago
